### PR TITLE
add MovieSection,some changes to moviecards

### PR DESCRIPTION
**Pull request by azmat239**

### CODE REVIEW

File: app/components/MovieSection/MovieSection.types.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { MovieCardtypes } from "../MovieCard/MovieCard.types";
+
+export type MovieSectionTypes = {
+  title: string;
+  tabs?: Array<{ tabsName: string; onClick?: React.MouseEventHandler }>;
```
Comment (UmmarHamid) [inline app/components/MovieSection/MovieSection.types.ts]:
```suggestion
  tabs?: Array<{ tabName: string; onClick?: React.MouseEventHandler }>;
```

File: app/components/MovieSection/MovieSection.types.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  movieLists: Array<MovieCardtypes>;
```
Comment (UmmarHamid) [inline app/components/MovieSection/MovieSection.types.ts]:
```suggestion
  moviesList: Array<MovieCardtypes>;
```

File: app/components/MovieSection/MovieSection.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                date={movieList.date}
+                onClick={movieList.onClick}
+                title={movieList.title}
+                MenuPopoverLinks={movieList.MenuPopoverLinks}
+              />
+            </div>
+          );
+        })}
+      </div>
+    </div>
```
Comment (UmmarHamid) [inline app/components/MovieSection/MovieSection.tsx]:
```suggestion
  const moviesClasses = classNames("flex flex-col mt-6");
  return (
    <div className={`${moviesClasses} ${extraClasses ?? ""}`}>
      <div className="flex gap-6 mx-4">
        <Heading text={title} variant="24" extraClasses="font-bold" />
        {tabs && (
          <div className="border-2 border-darkBlue rounded-30">
            {tabs.map((tab, index) => {
              return (
                <button
                  key={index}
                  onClick={tab.onClick}
                  className="py-1 px-5 active:bg-darkBlue rounded-30 "
                >
                  <Text
                    text={tab.tabsName}
                    variant="16"
                    extraClasses="active:text-transparent bg-clip-text active:bg-gradient-to-b active:from-lightGreen active:to-lightBlue font-semibold"
                  />
                </button>
              );
            })}
          </div>
        )}
      </div>
      <div className="flex my-6  hide-scrollbar overflow-x-auto">
        {movieLists.map((movieList, index) => {
          return (
            <div key={index}>
              <MovieCard
                image={movieList.image}
                rating={movieList.rating}
                date={movieList.date}
                onClick={movieList.onClick}
                title={movieList.title}
                MenuPopoverLinks={movieList.MenuPopoverLinks}
              />
            </div>
          );
        })}
      </div>
    </div>
```

File: app/globals.css [hunk 4]
```diff
@@ -2,26 +2,10 @@
...
-body {
-  color: rgb(var(--foreground-rgb));
-  background: linear-gradient(
-      to bottom,
-      transparent,
-      rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
+.hide-scrollbar::-webkit-scrollbar-thumb {
+  background-color: transparent; /* Hide the thumb */
```
Comment (UmmarHamid) [inline app/globals.css:10]: We shouldn't need these extra css styles, I wonder why they are required!

**Review by UmmarHamid — CHANGES_REQUESTED** [review]
Nice Work I like how you are able to comprehend React now.

Small feedback

File: app/globals.css (hunk 4 above)
Comment (azmat239) [reply app/globals.css:10]: Inorder to hide the scrollbar the predefined one wasn't able to solve

**Review by UmmarHamid — APPROVED** [review]

**Pull request merged**